cluster-controller/src/main/java/io/strimzi/controller/cluster/ClusterController.java

```diff
@@ -208,6 +208,7 @@ private void recreateConfigMapWatch() {
                 configMapWatch = res.result();
             } else {
                 log.error("Failed to recreate ConfigMap watch in namespace {}", namespace);
+                vertx.close();
```

Comment:
Actually, it might just be worth a comment about why closing vertx is the right thing to do here.

Reply:
Done - comment added.